features/issues/api/use-issues.tsx

```diff
@@ -4,17 +4,29 @@ import axios from "axios";
 import type { Page } from "@typings/page.types";
 import type { Issue } from "../types/issue.types";
 
-async function getIssues(page: number) {
+// * Filters defined here are NOT affecting test failures in pagination and reload
+
+export interface Filters {
+  status: string | null;
+  level: string | null;
+  project: string | null;
+}
+
+async function getIssues(page: number, filters: Filters) {
+  const { status, level, project } = filters;
+
   const { data } = await axios.get(
-    `https://prolog-api.profy.dev/issue?page=${page}`
+    `https://prolog-api.profy.dev/issue?page=${page}${
```

Comment:
Instead of the ternaries here you can use the second parameter of `axios.get`. Here an example:

```
// Equivalent to `axios.get('https://httpbin.org/get?answer=42')`
await axios.get('https://httpbin.org/get', { params: { answer: 42 } });
```

I took this example from this blog post: https://masteringjs.io/tutorials/axios/get-query-params

Reply:
This is significantly cleaner, great pick.